lib/features.js

```diff
@@ -14,6 +14,12 @@ const packageHelper = require('./package-helper');
 /**
  * An object that holds internal configuration about different
  * "loaders"/"plugins" that can be enabled/used.
+ *
+ * @type {{[key: string]: {
+ *     method: string,
+ *     packages: Array<{ name: string, version?: string } | Array<{ name: string }>>,
```

Comment:
this does not make sense. The proper type is `Array<{ name: string, enforce_version?: boolean }>`

Reply:
See https://github.com/symfony/webpack-encore/blob/main/lib/features.js#L23, `Array<{ name: string }>` is like "install one of those dependencies"

Reply:
an, I misread the nesting.

Still, the `version` key does not make sense as the optional key we have is `enforce_version`